frontend/components/distribution/TableItems.vue

```diff
@@ -0,0 +1,34 @@
+<template>
+  <v-container fluid grid-list-md>
+    <v-data-iterator :items="distributed" content-tag="v-layout" row wrap hide-actions>
+      <template v-slot:item="props">
+        <v-flex xs3 sm6 md4 lg3>
```

Comment:
```suggestion
        <v-flex xs6 sm6 md3 lg3>
```
it should always display an even numbers of cards per line (in our case 2 or 4), but never 3.
As the grid is 12 in total, only 3 and 6 are good value.